hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -304,6 +309,8 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6}>
             <OverviewCard loading={loading} title='Deleted Containers' data={deletedContainers.toString()} icon='delete' />
           </Col>
+        </Row>
+        <Row gutter={[25, 25]}>
```

Comment:
We don't need to separate out into rows.
Single row parent wrapper would automatically handle card breaks to new rows.
Could you check once without this <Row> component wrapped?

Reply:
Done changes in latest 2 commits.